scripts/run_container_bash.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+SRC_DIR=${SRC_DIR:-`pwd`}
```

Comment:
Can this script be merged with run_container_jupyter.sh? Theyre so similar it seems itd be easier to maintain one, rather than two

Reply:
Actually can all three bash scripts be merged? One with a nice CLI would be better for both users and maintenance

Reply:
Yes, I have combine them into one script, and also update the Docker documents.